tsconfig.json

```diff
@@ -43,6 +43,7 @@
     "noUnusedParameters": true,
     "noUnusedLocals": true,
     "skipLibCheck": true,
+    "resolveJsonModule": true,
```

Comment:
Cool feature I turned on in Typescript [that automatically types values from imported JSON files](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-9.html).

Reply:
🔥